sample_install-deps.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+## Modify the following lines to install dependencies for your project.
+echo "Installing dependencies for non-ROS packages"
+apt update -q
+apt install -y --no-install-recommends git libgoogle-glog-dev \
+    libgflags-dev \
+    libatlas-base-dev \
+    libeigen3-dev \
+    libsuitesparse-dev
+
+mkdir -p ./deps && cd ./deps
+git clone https://ceres-solver.googlesource.com/ceres-solver
```

Comment:
Consider cloning with specifying the folder git goes into as shown here  (i have 
 never tried it before) https://stackoverflow.com/questions/651038/how-do-i-clone-a-git-repository-into-a-specific-folder
 
 This makes the folder it goes into explicit.
 
 Also, specifying the commit / tag to build from would be good. if not when master is updated, we get affected.